### ISSUE TITLE
Notification classes missing in bootstrap theme's scss file

### ISSUE THREAD

**Issue by feng-xiao** [telerik/kendo-angular#3493]
**Describe the bug**
Notification css class is not defined in dist/all.scss

**To Reproduce**
- install kendo-angular-notification@3.0.3, kendo-theme-bootstrap@4.42.0
- import node_modules/@progress/kendo-theme-bootstrap/dist/all.scss
- display error, success, etc. notification
- background color is not displayed correctly due to css class for respective notification type is missing (.e.g. k-notification-error)

**Expected behavior**
Notification is shown in correct background according to specified type

**Screenshots**
The notification is shown in white background color
![image](https://user-images.githubusercontent.com/3407775/140687146-4021b26e-d2ea-4d28-8748-66f5fa7e0cd8.png)

**Browser**
 - OS: Windows 11
 - MS Edge latest 

**Additional context**

**Comment by Raisolution** [comment]
Hi @feng-xiao 

We were not able to reproduce the mentioned behavior.

The mentioned versions of the Notification package and the Bootstrap theme are used in our documentation
https://www.telerik.com/kendo-angular-ui/components/notification/types/
here is a stackblitz example of the same
https://stackblitz.com/edit/angular-xnpwvx?file=index.html

If further assistance with implementing this configuration is needed, please open a [support ticket](https://www.telerik.com/account/support-tickets).

**Comment by Godrules500** [comment]
I am having the same issue.  I am on version ^4.42.0.

Every theme works except error for me. I inspected the html and the k-notification-error css is not being applied (doesn't even show up in the css classes)

**Comment by feng-xiao** [comment]
Hi @Raisolution, where's variable $notification-themes defined? I can't find it in the library. It's being used in the following code, I think that's why no background color is showing the notifications.

```scss
@each $theme, $props in $theme-colors {
    $notification-themes: map-merge($notification-themes, ($theme: notification-theme($theme, $props)) );
}
```

**Comment by roman-savitski** [comment]
> Hi @Raisolution, where's variable $notification-themes defined? I can't find it in the library. It's being used in the following code, I think that's why no background color is showing the notifications.
> 
> ```css-scss
> @each $theme, $props in $theme-colors {
>     $notification-themes: map-merge($notification-themes, ($theme: notification-theme($theme, $props)) );
> }
> ```

There are 3 definitions of `$theme-colors` in all.scss
```
$theme-colors: (
    "primary": $primary,
    "secondary": $secondary,
    "tertiary": $tertiary,
    "info": $info,
    "success": $success,
    "warning": $warning,
    "error": $error,
    "dark": $dark,
    "light": $light,
    "inverse": $inverse
) !default;

....  little below ....

$theme-colors: (
  "primary":    $primary,
  "secondary":  $secondary,
  "success":    $success,
  "info":       $info,
  "warning":    $warning,
  "danger":     $danger,
  "light":      $light,
  "dark":       $dark


...... little below ......


$theme-colors: (
    "primary": $primary,
    "secondary": $secondary,
    "tertiary": $tertiary,
    "info": $info,
    "success": $success,
    "warning": $warning,
    "error": $error,
    "dark": $dark,
    "light": $light,
    "inverse": $inverse
) !default;

```

For some reason 2nd definition is used, and 3rd definition is ignored.
Have no idea why there are 3 definitions and why 3rd one is not used as default.

Its compiled correctly in `https://kendo.cdn.telerik.com/themes/4.42.0/bootstrap/bootstrap-main.css` thats why this is working on slackbitz. But if imported in Angular app like `@import "../../node_modules/@progress/kendo-theme-bootstrap/dist/all.scss";` -> incorrect CSS is produced

**Comment by roman-savitski** [comment]
> Hi @feng-xiao
> 
> We were not able to reproduce the mentioned behavior.
> 
> The mentioned versions of the Notification package and the Bootstrap theme are used in our documentation https://www.telerik.com/kendo-angular-ui/components/notification/types/ here is a stackblitz example of the same https://stackblitz.com/edit/angular-xnpwvx?file=index.html
> 
> If further assistance with implementing this configuration is needed, please open a [support ticket](https://www.telerik.com/account/support-tickets).

@Raisolution This issue is reproducible when building Angular app using .scss files (not .css).

 `kendo-theme-bootstrap/modules/bootstrap/scss/_variables.scss` has variable:
```
$theme-colors: (
  "primary":    $primary,
  "secondary":  $secondary,
  "success":    $success,
  "info":       $info,
  "warning":    $warning,
  "danger":     $danger,
  "light":      $light,
  "dark":       $dark
) !default;
```

For some reason this variable is not overrided by `_bootstrap-overrides.scss`, as result "error" style is not included into final CSS.

**Comment by JoomFX** [comment]
Hi @feng-xiao , @roman-savitski , @Godrules500 ,

I tried to reproduce the issue with a local Angular application (where I imported the SCSS file) and with [this Stackblitz example](https://stackblitz.com/edit/angular-tguh3e-lrczcq?file=app/app.component.scss) (where I also imported the SCSS file), but I couldn't.

Please make sure that our Kendo Bootstrap Theme is imported before the Bootstrap theme, as described [here](https://www.telerik.com/kendo-angular-ui/components/styling/theme-bootstrap/#toc-installation).

If the above does not solve the issue, please open a [support ticket](https://www.telerik.com/account/support-tickets) and provide a sample app where we can see the problem and investigate it further.

**Comment by feng-xiao** [comment]
Hi @JoomFX , the difference between your code with mine is that yours is importing kendo scss inline with component *styleUrl* property. I'm webpack/gulp-sass to build css file from scss.
![image](https://user-images.githubusercontent.com/3407775/143286007-e7c881eb-a8e5-4805-8675-bfb1ca14c918.png)

**Comment by feng-xiao** [comment]
Hi @JoomFX , @Raisolution , I tried import the kendo ui (all.scss) in the style file specified in angular.json, it also doesn't work. Please note I didn't have issues with version before 4.42.0. Not sure what new changes break it.

**Comment by Raisolution** [comment]
Hi @feng-xiao ,

Please open a [support ticket](https://www.telerik.com/account/support-tickets) so we could provide further assistance with resolving the problem.

**Comment by feng-xiao** [comment]
> Hi @feng-xiao ,
> 
> Please open a [support ticket](https://www.telerik.com/account/support-tickets) so we could provide further assistance with resolving the problem.

Done

**Comment by feng-xiao** [comment]
Hi @JoomFX , @Raisolution, @roman-savitski ,

I found why the new version of Kendo UI scss doesn't work, it's because it's missing **map-merge** call. I think Kendo UI should add the logic back to avoid breaking changes. I already replied to Kendo UI support in my ticket regarding my findings.

![image](https://user-images.githubusercontent.com/3407775/143455871-45217f4d-eb2b-453a-9b74-d71d2d86802a.png)
